### ISSUE TITLE
Embedded artwork in locally added files no longer extracted

### ISSUE THREAD

**Issue by ecronin** [Automattic/pocket-casts-android#2122]
### Description

Somewhere in the recent episode artwork refactors the Android app lost the ability to extract and display the embedded artwork from custom episode files added from phone storage (bottom file in screenshot). Episodes added using the web interface still have embedded artwork extracted and displayed on the phone once synced, although once the synced file is deleted from the cloud the on-phone copy loses its artwork in a different way (top file in screenshot). Episodes which were added before the artwork refactor, either on the phone (never synced to the cloud) or through the website (whether or not they're still stored in the cloud) still display thier extracted artwork.

In #1987 the setting was changed from always true to always false for extracting the embedded artwork from the metadata of custom episodes added from local storage on the phone: [AddFileActivity.kt](https://github.com/Automattic/pocket-casts-android/commit/7e46d20ffadf0152544307e06f8161063c3e3caf#diff-75f467e711b7dfc03870b1f8c651014e59a6c780343fde60836753b17a6c2da4) which I think might be where extracting the artwork on phone broke. A quick look through the change wasn't enough to spot how the storage of custom episode artwork once it had been extracted had changed.

### Step-by-step reproduction instructions

1. Copy an mp3 with embedded artwork to your phone
2. Open Files, note that Files shows the artwork in the file thumbnail
3. Share or Open With the mp3 file and select Pocket Casts
4. Note that Pocket Casts shows a generic file icon when adding and once added
5. Open the web player in a browser
6. Drag another mp3 with embedded artwork into the browser to upload it
7. Note that after uploading the browser shows the embedded artwork
8. Sync the phone and download the new cloud episode
9. Note that the synced file has artwork in the Files list and on playback
10. Select the file on the phone and remove from the cloud
11. Note that the file no longer has artwork in the Files list or on playback, but does have a small thumbnail on file details

### Screenshots or screen recording

![Screenshot_20240427-133949](https://github.com/Automattic/pocket-casts-android/assets/21134732/ab7cb21a-e6ed-41f0-a4ae-938b1eb2286a)

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Pocket Casts 7.62-rc-5

**Comment by MiSikora** [comment]
Thank you for the detailed description. I can confirm the bug with a missing artwork while uploading the file.

Do you have `Use episode artwork` setting enabled? It is under `Appearance` section in the settings.

**Comment by ecronin** [comment]
I usually have 'use episode artwork' disabled. I had previously experimented if toggling it fixed adding files and it didn't. I just toggled it and it does display the artwork for the top case in the screenshot, and stops displaying it if I toggle again. Episodes that had been added to my phone prior to the episode artwork changes do not need 'use episode artwork' enabled to display.  Because there's no meaningful feed artwork for local files, always displaying the episode artwork if present makes more sense to me.
